pom.xml

```diff
@@ -67,7 +67,7 @@
 			<resource>
 				<directory>src</directory>
 				<includes>
-					<include>resources/*.png</include>
+					<include>resources/**/*</include>
```

Comment:
Funny thing is that I never noticed this because it seems to add it anyways *(at least on my end)*. Nice catch.